region/maridia/inner-pink.json

```diff
@@ -7373,20 +7388,62 @@
           "from": 5,
           "to": [
             {
-              "id": 3,
+              "id": 4,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    "Gravity",
+                    {"or":[
+                      "SpaceJump",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      "h_canDiagonalBombJump",
+                      {"and":[
+                        "h_canJumpIntoIBJ",
+                        "canIframeSpikeJump",
+                        {"spikeHits": 1}
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Cacatac Alley Springball Spike Boost",
+                  "notable": true,
+                  "requires": [
+                    "HiJump",
+                    "canSuitlessMaridia",
+                    "canCarefulJump",
+                    "canSpringBallJumpMidAir",
+                    "canLateralMidAirMorph",
+                    "canNeutralDamageBoost",
+                    { "enemyDamage": { "enemy": "Cacatac", "hits": 2, "type": "spike" } }
+                  ],
+                  "note": "When the Cacatac on the ground fires a spike, perform a  springball Jump to break the waterline and then hit the spike for extra height."
+                },
+                {
+                  "name": "Underwater Walljump",
+                  "notable": false,
+                  "requires": [
+                    "canUnderwaterWalljump",
+                    "canSpaceJumpWaterBounce"
+                  ]
+                }
+              ]
+            },
+            {
+              "id": 6,
               "strats": [
                 {
                   "name": "Base",
                   "notable": false,
                   "requires": [
                     "Gravity",
-                    "Morph",
                     {"or": [
                       "SpaceJump",
-                      "canCarefulJump",
-                      "h_canResetFallSpeed",
-                      "canSuitlessMaridia",
                       "h_canHBJ",
+                      "h_canResetFallSpeed",
```

Comment:
That's what I was thinking as well, but then i realized it can be used to make the jump with gravity from the bottom as well. jump and midair morph and press against the left wall, then unmorph after to move left a bit more